### PR TITLE
Fix Pylint warnings

### PR DESCRIPTION
**Pull request by cofri**
Pylint returns relevant warnings which are not raised by flake8, the linter used in our CI process. This PR aims at fixing some of these errors to improve deel-lip base code. Each commit of this PR deals with one message type, given in the commit message.

Descriptions for all Pylint messages can be found [here](https://pylint.pycqa.org/en/latest/user_guide/messages/messages_overview.html).

### CODE REVIEW

File: deel/lip/layers/pooling.py
Comment (thib-s) [inline deel/lip/layers/pooling.py:1]: We need to check if the argument `training` can be removed. ( especially when calling `model(x, training=True)` )
(this behavior might be dependent on the tf version)

**Review by thib-s — COMMENTED** [review]
Great work, we need to discuss the previous comment, otherwise LGTM

File: deel/lip/layers/pooling.py
Comment (y-prudent) [reply deel/lip/layers/pooling.py:1]: Maybe we should keep `**kwargs` in the call when `training` is not used?

**Review by y-prudent — COMMENTED** [review]
Same remark as Thibaut, once the issue (?) is solved LGTM

**Review by thib-s — APPROVED** [review]
I've been looking at the code of official keras layers, the parameter `training` seems not mandatory when the layer has the same behavior in train and test.

**Review by franckma31 — APPROVED** [review]
fine for me

**Pull request merged**